GameData/Mkerb/MkerbInc/Localization/sw-sw.cfg

```diff
@@ -0,0 +1,458 @@
+		#MKERB-Sound-KerbinFlyingHigh = As the air pressure drops, your ability to get data diminishes severely.
+		#MKERB-Sound-KerbinInSpaceLow = Recorded data is very strange - still showing a very, very small pressure wave produced. How could there be pressure waves in a vacuum?
+		#MKERB-Sound-KerbinInSpaceHigh = There might be some unseen particles in the vacuum of space.
+		#MKERB-Sound-MunSrfLanded = Nothing happens, of course.
+		#MKERB-Sound-MunFlyingLow = Amounts of the newly detected particle are definitely decreasing as we get further into the Mun's gravity well.
+		#MKERB-Sound-MunFlyingHigh = Based on this data, scientists at KSC are theorizing that this may be a new type of particle.
+		#MKERB-Sound-MinmusInSpace = This data shows that there may be a new type of particle
+		#MKERB-Sound-MinmusSrfLanded = The scientists at KSC have not yet found a name for the new particle.
+		#MKERB-Sound-InSpace = We're no longer able to track anything on the new particle. 
+
+		#MKERB-Sound-GillyInSpace = So small you could swallow it and make sound training with it.
+		#MKERB-Sound-GillySrfLanded = You don't even bother.
+
+		#MKERB-Sound-NeidonInSpace = No one will hear your screams!
+		#MKERB-Sound-NeidonSrfLanded = You are scared that you will never hear another kerbal.
+
+		#MKERB-Sound-PlockInSpace = You don't belive in sound anymore
+		#MKERB-Sound-PlockSrfLanded = Nice echos but noone to hear them.
+
+		#MKERB-Sound-PoltaInSpace = You scream against the giant rock.
+		#MKERB-Sound-PoltaSrfLanded = You bang to rocks together, nothing happens.
+
+		#MKERB-Sound-TalInSpace = The sound seems to flow around it.
+		#MKERB-Sound-TalSrfLanded = You stomp your foot and nearly fly off into space.
+
+		#MKERB-Sound-TektoInSpace = Looks like a sound chamber.
+		#MKERB-Sound-TektoSrfLanded = You try to make a sound but nothing come out of your mouth.
+
+		#MKERB-Sound-LaytheInSpace = Water sounds. How you miss them!
+		#MKERB-Sound-LaytheSrfLanded = You hear some water or did you just dream that ?
+
+		#MKERB-Sound-EelooInSpace = Looks like a gigant ice berg.
+		#MKERB-Sound-EelooSrfLanded = Its to cold to press the button.
+
+		#MKERB-Sound-ErnusInSpace = The heat does generate intresting data.
+		#MKERB-Sound-ErnusSrfLanded = The vribartions are too severe.
+
+		#MKERB-Sound-AtellInSpace = Quiet place.
+		#MKERB-Sound-AtellSrfLanded = The sound seems to be sucked away.
+
+		#MKERB-Sound-ApturInSpace = Bill would like to sind in th shower.
+		#MKERB-Sound-ApturSrfLanded = The microphone is working.
+
+		#MKERB-Sound-LaveInSpaceHigh = You hear a strange sqeek noise.
+		#MKERB-Sound-LaveSrfLanded = The microphone got wet.
+
+		#MKERB-Sound-TitanusInSpace = The device freaks out. Maybe jeb did play with it.
+		#MKERB-Sound-TitanusSrfLanded = You hear some movement under the ground.
+
+		#MKERB-Sound-VanorInSpace = The dust in the device makes the data unreliable.
+		#MKERB-Sound-VanorSrfLanded = It seems to pick up some data.
```

Comment:
@chakra2017 would you also translate these strings?

Reply:
ok